npm/npm.go

```diff
@@ -77,6 +80,58 @@ func (npMgr *NetworkPolicyManager) GetClusterState() telemetry.ClusterState {
 	return npMgr.clusterState
 }
 
+// SendAiMetrics :- send NPM metrics using AppInsights
+func (npMgr *NetworkPolicyManager) SendAiMetrics() {
+	var (
+		aiConfig = aitelemetry.AIConfig{
```

Comment:
i dont think everything has to be in var..just declare fields and define them outside of var..thats usual practice right?

Reply:
for init blocks I use var, if it was only a couple or single init I use := || var (name) (type)

Reply:
why dont we move outside of var? thats the standard we follow all along.